tests/integration/test_plugins.py

```diff
@@ -24,6 +24,35 @@
 PG_TRGM_EXTENSION_STATEMENT = "SELECT word_similarity('word', 'two words');"
 PLPYTHON3U_EXTENSION_STATEMENT = 'CREATE FUNCTION plpython_test() RETURNS varchar[] AS $$ return "hello" $$ LANGUAGE plpython3u;'
 UNACCENT_EXTENSION_STATEMENT = "SELECT ts_lexize('unaccent','Hôtel');"
+BLOOM_EXTENSION_STATEMENT = (
```

Comment:
We can add those in a `plugin`: `SQL test` dictionary and then loop over `.keys()`  in the config definition on [L#192](https://github.com/canonical/postgresql-k8s-operator/blob/bb523a2946c1ddae55e70f219e0da6a1dc0a4dfd/tests/integration/test_plugins.py#L193) and over `.values()` in the checks  on [L#74](https://github.com/canonical/postgresql-k8s-operator/blob/bb523a2946c1ddae55e70f219e0da6a1dc0a4dfd/tests/integration/test_plugins.py#L74) and [L#232](https://github.com/canonical/postgresql-k8s-operator/blob/bb523a2946c1ddae55e70f219e0da6a1dc0a4dfd/tests/integration/test_plugins.py#L232).